keyboards/lazydesigners/bolt/info.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
This renders incorrectly (using `ctrl + shift + i` within configurator)
![image](https://user-images.githubusercontent.com/16520359/90324244-06153c00-df64-11ea-9239-7948abefadbb.png)

Reply:
Thanks! Update info.json with a clear layout.